### PR TITLE
Added documentation for R4 Coverage (Encounter level) Delete

### PR DESCRIPTION
**Pull request by bhargav-kvs**
Description
----
Added documentation for R4 Coverage (Encounter level) Delete.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screen Shot 2020-05-22 at 7 48 43 AM

### CODE REVIEW

File: content/millennium/r4/financial/coverage.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ _Implementation Notes_
 * This implementation follows the [JSON Patch](https://tools.ietf.org/html/rfc6902) spec.
 * Only operations on the paths listed below are supported.
 * Only Encounter-level Coverages may be patched. Patches are not currently supported for Patient-level Coverages.
+* The `id` is a combination of encounter id `123` and coverage id `456` separated by a hyphen. Example: `123-456`
```
[the remark this answers is not in the document]
Comment (bhargav-kvs) [reply content/millennium/r4/financial/coverage.md]: removed https://github.com/cerner/fhir.cerner.com/pull/386/commits/684b58cb3b35c4a82960eb59371309afa4d14203#diff-81b4004b2ae7fd9a288412597630810cL189

File: content/millennium/r4/financial/coverage.md [hunk 2]
```diff
@@ -234,6 +235,51 @@ The `ETag` response header indicates the current `If-Match` version to use on su
...
+## Delete
+
+Delete an existing Encounter-level Coverage.
+
+    DELETE /Coverage/:id
+
+_Implementation Notes_
+
+* Only Encounter-level Coverages may be deleted. Deletes are not currently supported for Patient-level Coverages.
+* The `id` is a combination of encounter id `123` and coverage id `456` separated by a hyphen. Example: `123-456`
```
[the remark this answers is not in the document]
Comment (bhargav-kvs) [reply content/millennium/r4/financial/coverage.md]: removed https://github.com/cerner/fhir.cerner.com/pull/386/commits/684b58cb3b35c4a82960eb59371309afa4d14203#diff-81b4004b2ae7fd9a288412597630810cL247

**Comment by bhargav-kvs** [comment]
Updated the screenshot as well

File: content/millennium/r4/financial/coverage.md [hunk 3]
```diff
@@ -234,6 +234,50 @@ The `ETag` response header indicates the current `If-Match` version to use on su
...
+
+    DELETE https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Coverage/623884-674012
+
+#### Response
+
+<%= headers status: 204 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
```
Comment (MaxPhilips) [inline content/millennium/r4/financial/coverage.md:267]: i don't see content-length or content-type coming back from the server

Comment (bhargav-kvs) [reply content/millennium/r4/financial/coverage.md]: removed content-length and content-type. Also updated the screenshot in the PR description https://github.com/cerner/fhir.cerner.com/pull/386/commits/160ac7c1555048d193378a0b5d944e863b8e7988#diff-81b4004b2ae7fd9a288412597630810cL266

**Comment by MaxPhilips** [comment]
verified on live

Screen Shot 2020-05-22 at 3 30 46 PM

**Pull request merged**